docs/zephyr-integration.md

```diff
@@ -0,0 +1,493 @@
+# Zephyr Test Management Integration
```

Comment:
by comparing with other docs you could see there some differences with your code... like the logo image is not given as the first line, like api-hub and bugsnag have.
Moreover If we create this page then we should it make it visible by modifying the `manifest.json` so it is added at the same way that other {product}-integration.md pages.

Reply:
Also, the file needs to be moved to the directory `docs/products/SmartBear MCP Server` I suppose